doc/source/tune/api/trainable.rst

```diff
@@ -174,23 +174,6 @@ declare the resources per trial based on the given configuration.
 
 It is also possible to specify memory (``"memory"``, in bytes) and custom resource requirements.
 
-
-.. _tune-function-docstring:
-
-session (Function API)
-----------------------
-
-.. currentmodule:: ray
-
-.. autosummary::
-
-    air.session.report
-    air.session.get_checkpoint
-    air.session.get_trial_name
-    air.session.get_trial_id
-    air.session.get_trial_resources
-    air.session.get_trial_dir
-
```

Comment:
Do you think it would be helpful for tune users to have Train context/utilities referenced somewhere in this page?

https://docs.ray.io/en/master/train/api/api.html#ray-train-utilities